kano-validation.js

```diff
@@ -41,11 +41,17 @@ export const Mixin = (superClass) => class extends superClass {
         this.set('errors.username', undefined);
         return true;
     }
-    validatePassword (password) {
+    validatePassword(password) {
         if (!password) {
             this.set('errors.password', "Password cannot be empty.");
             return false;
         }
+        
+        password.trim();
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/Trim#Description

That won't mutate the string but return a new one

Reply:
good point, have adjusted